apis/oam/v1alpha2/core_workload_types.go

```diff
@@ -162,7 +170,13 @@ type ContainerEnvVar struct {
 	Name string `json:"name"`
 
 	// Value of the environment variable.
-	Value string `json:"value"`
+	// +optional
+	Value string `json:"value,omitempty"`
```

Comment:
Should this be a string pointer now?

Reply:
This is what brought up my question in Slack. I opted against it because it is not a pointer here: https://github.com/kubernetes/api/blob/eb909d5fe0e78daacf8a37e5c7e2414dd4d2e11b/core/v1/types.go#L1830

However, I do believe that a pointer is more conformant with api standard. What do you think?